packages/auth-providers/dbAuth/setup/src/webAuthn.setupData.ts

```diff
@@ -8,10 +8,10 @@ import { functionsPath, libPath } from './setupData'
 export { extraTask } from './setupData'
 
 // required packages to install on the web side
-export const webPackages = ['@simplewebauthn/browser']
+export const webPackages = ['@simplewebauthn/browser@^6']
 
 // required packages to install on the api side
-export const apiPackages = ['@simplewebauthn/server']
+export const apiPackages = ['@simplewebauthn/server@^6']
```

Comment:
These may be pinned to v7 if I got https://github.com/redwoodjs/redwood/pull/7477 working